src/Service/Models/GraphQLFilterParsers.cs

```diff
@@ -542,7 +543,7 @@ public static class FieldFilterParser
                 predicates.Push(new PredicateOperand(new Predicate(
                     new PredicateOperand(column),
                     op,
-                    new PredicateOperand(processLiteral ? $"{processLiterals(value)}" : value.ToString()))
+                    new PredicateOperand(processLiteral ? $"{processLiterals(value, column.ColumnName)}" : value.ToString()))
```

Comment:
why does this need a columnName as input?

Reply:
Because need to determine the DbType using the metadata that we are storing in `ColumnDefinition.DbType `inside every `SourceDefinition`.